sympy/parsing/sym_expr.py

```diff
@@ -46,14 +46,17 @@ class SymPyExpression(object):
 
         Example of parsing C code:
 
+        >>> from sympy.parsing.sym_expr import SymPyExpression
         >>> src = '''
         ... int a,b;
         ... float c = 2, d =4;
         ... '''
         >>> a = SymPyExpression(src, 'c')
         >>> a.return_expr()
-        [Declaration(Variable(Symbol('a'), type=IntBaseType(String('integer')), value=Integer(0))), Declaration(Variable(Symbol('b'), type=IntBaseType(String('integer')), value=Integer(0))), Declaration(Variable(Symbol('c'), type=IntBaseType(String('integer')), value=Integer(2))), Declaration(Variable(Symbol('d'),     type=IntBaseType(String('integer')), value=Integer(4)))]
-
+        [Declaration(Variable(a, type=integer, value=0)),
+        Declaration(Variable(b, type=integer, value=0)),
+        Declaration(Variable(c, type=integer, value=2)),
+        Declaration(Variable(d, type=integer, value=4))]
```

Comment:
I've corrected the doctest to match what is actually printed but I don't think it's actually correct. It looks to me like there should be quotes around `a`, `integer` etc and also I think `c` and `d` should of type `float` rather than `integer`.